<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorArea</class>
 <widget class="QWidget" name="MonitorArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#switch_lamp,QPushButton#switch_humi,QPushButton#switch_fan,QPushButton#switch_curtain_positive,QPushButton#switch_curtain_reverse
{
	border-image: url(:/images/switch_off.png);
}

/*{
	background-color: rgb(120, 120, 220);
	border-style: solid;
	border-radius: 25px;
	border-width: 2px;
	border-color:rgb(255, 255, 0);


}
*/
QWidget#page_monitor,QWidget#page_control
{
		
	background-color: rgb(200, 255, 170);
	border-color: rgb(85, 0, 0);
	border-width:1px;
	border-style:solid;

}

QPushButton#btn_aid,QPushButton#btn_aid_2
{
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(0, 0, 127);
	background-color: transparent;
	border-radius:10px;
}
QPushButton#btn_aid:pressed,QPushButton#btn_aid_2:pressed
{
	
	background-color: rgb(150, 255, 80);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_monitor">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>48</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>空气湿度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>164</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CO2浓度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>光照强度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>空气温度</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_to_control">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>40</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/add.ico);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_zids">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>79</y>
         <width>70</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>土壤湿度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_temp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-℃</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_humi">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>48</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-%</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_lux">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-lux</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_co2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>164</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-ppm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_humi_soil">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-%</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_aid">
       <property name="geometry">
        <rect>
         <x>172</x>
         <y>2</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1区</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_control">
      <widget class="QPushButton" name="switch_lamp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="switch_humi">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="switch_curtain_positive">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="switch_fan">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;帘&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;灯&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;水&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;风&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_aid_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>2</y>
         <width>50</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1区</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_to_monitor">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>-7</y>
         <width>40</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>设置节点</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/add.ico);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="switch_curtain_reverse">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>35</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>34</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
